CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - ???
+
+### Added
+
+- Allow parsing to throw exceptions
+
+### Changed
+
+- Moved CSV parsing to a separate package
```

Comment:
Should we put a link to the new package?

Reply:
It's linked in the readme.

Reply:
ah, ok